framework/widgets/ActiveField.php

```diff
@@ -158,6 +163,11 @@ class ActiveField extends Component
      */
     private $_skipLabelFor = false;
 
+    /**
+     * @var bool whether `aria-invalid` HTML attribute should be updated by client validation
```

Comment:
why should that be necessary? If client validation finds that the error is gone it should update it, no?